lumigator/backend/backend/services/jobs.py

```diff
@@ -430,34 +514,23 @@ def create_job(
 
         # ...and use directly Job*Config(request.job.config.model_dump_json())
         job_config_args = {
-            "--config": job_config.model_dump_json(exclude_unset=True, exclude_none=True),
+            "--config": job_config.model_dump_json(),
```

Comment:
hm, we back to removing this lol

Reply:
Sorry!!!! I'll put it as it is in main T_T
Git being git....